sources/packages/backend/apps/api/src/route-controllers/supporting-user/models/supporting-user.dto.ts

```diff
@@ -25,10 +28,29 @@ export class ApplicationIdentifierAPIInDTO {
   @Length(APPLICATION_NUMBER_LENGTH, APPLICATION_NUMBER_LENGTH)
   applicationNumber: string;
   @IsNotEmpty()
-  studentsDateOfBirth: string;
-  @IsNotEmpty()
   @MaxLength(USER_LAST_NAME_MAX_LENGTH)
   studentsLastName: string;
+  @IsEnum(SupportingUserType)
+  supportingUserType: SupportingUserType;
+  /**
+   * For Parent search only. Required for Parent, Optional for Partner.
+   */
+  @ValidateIf(
+    (object: UpdateSupportingUserAPIInDTO) =>
+      object.supportingUserType === SupportingUserType.Parent,
+  )
```

Comment:
Sorry about the confusion, the object must be typed with the DTO type, which in this case and on line 49 as well is  `ApplicationIdentifierAPIInDTO`


```ts
@ValidateIf(
    (object: ApplicationIdentifierAPIInDTO) =>
      object.supportingUserType === SupportingUserType.Parent,
  )
```